oolab/src/main/java/agh/ics/oop/RectangularMap.java

```diff
@@ -19,7 +20,7 @@ public boolean canMoveTo(Vector2d dest){
 
     @Override
     public Object objectAt(Vector2d position) {
-        return animals.stream()
+        return animals.values().stream()
```

Comment:
tu nie korzystamy w ogóle z potu nie wykorzystujemy potencjału mapy - po to ją mamy żeby nie przeszukiwać wartości tylko użyć `get()` ;)